makerspace/views.py

```diff
@@ -0,0 +1,84 @@
+from django.views.generic import ListView, DetailView, UpdateView, CreateView, DeleteView, TemplateView
+from makerspace.models import Makerspace, Tool
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.urls import reverse_lazy
+
+
+
+class ViewMakerspaceView(TemplateView):
+    template_name = 'makerspace/makerspace.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['latest_makerspace_announce'] = Makerspace.objects.last()
+        print("QP", context)
+        print(context['view'])
+        return context
+
+
+class ViewAdminMakerspaceView(PermissionRequiredMixin, UpdateView):
+    model = Makerspace
+
+    template_name = 'makerspace/admin_makerspace.html'
+    context_object_name = 'makerspace'
+    permission_required = 'makerspace.add_Makerspace'
+    fields = (
+        'title',
```

Comment:
If we are using all fields, use `fields = "__all__"` instead. This goes for the other views below as well.